src/app/components/layout/InfoPanel.scss

```diff
@@ -33,6 +33,14 @@
       @include font-size(14px);
     }
 
+    .lmi {
+      font-size: 14px;
+
+      @include desktop-down {
+        @include font-size(18px);
```

Comment:
The CMI font continues to change as the displays  get smaller (tablet-down 17px and phone-down 16px) while LMI remains 18px. Add missing media queries for LMI.

Reply:
Done.